tests/integration.rs

```diff
@@ -6176,7 +6035,7 @@ mod tests {
                 guest.wait_vm_boot(None).unwrap();
 
                 // Check if SGX is correctly detected in the guest.
-                assert!(guest.check_sgx_support().unwrap());
+                assert!(guest.check_sgx_support().is_ok());
```

Comment:
Asserting on a boolean like this loses the underlying error. Using `.expect()` or just `.unwrap()` would make the underlying error visible in the logs.

Reply:
Ah yes I understand that's indeed better. Let me unwrap all these.